packages/clay-multi-step-nav/stories/index.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import '@clayui/css/lib/css/atlas.css';
+import ClayMultiStepNav from '../src';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+const spritemap = require('@clayui/css/lib/images/icons/icons.svg');
+
+const ClayMultiStepNavWithState = () => {
+	const [value, setValue] = React.useState<number>(1);
+
+	const steps = [
+		{
+			active: value === 0,
+			complete: value > 0,
+			onClick: () => setValue(0),
+			subTitle: 'SubOne',
+			title: 'One',
+		},
+		{
+			active: value === 1,
+			complete: value > 1,
+			onClick: () => setValue(1),
+			subTitle: 'SubTwo',
+			title: 'Two',
+		},
+		{
+			active: value === 2,
+			complete: value > 2,
+			onClick: () => setValue(2),
+			subTitle: 'SubThree',
+			title: 'Three',
+		},
+		{
+			active: value === 3,
+			complete: value > 3,
+			onClick: () => setValue(3),
```

Comment:
Perhaps the use of `onClick` in each item can be removed from here and we pass it on the` map` with the `index` parameter.

Reply:
Since this is just the low-level API for now, I think it makes more sense to pass it into the items directly. This allows greater flexibility on the low-level API by allowing users to mix with steps with both `onClick` and `href`. Likely in the high-level implementation, we would pass it in with the `map` and `index` params though.

Reply:
Hmmm, I'm not so sure about this but I think we can go with that.